openedx/core/djangoapps/credit/api.py

```diff
@@ -70,45 +75,51 @@ def get_credit_requirements(course_key, namespace=None):
         namespace(str): Namespace of requirements
 
     Example:
-        >>> get_credit_requirements("course-v1-edX-DemoX-1T2015")
-                {
-                    requirements =
-                    [
-                        {
-                            "namespace": "reverification",
-                            "name": "i4x://edX/DemoX/edx-reverification-block/assessment_uuid",
-                            "display_name": "Assessment 1",
-                            "criteria": {},
-                        },
-                        {
-                            "namespace": "proctored_exam",
-                            "name": "i4x://edX/DemoX/proctoring-block/final_uuid",
-                            "display_name": "Final Exam",
-                            "criteria": {},
-                        },
-                        {
-                            "namespace": "grade",
-                            "name": "grade",
-                            "display_name": "Grade",
-                            "criteria": {"min_grade": 0.8},
-                        },
-                    ]
-                }
+        >>> get_credit_requirements
```

Comment:
minor: i think the example argument to the function, which was removed in this change, should be restored.